### PR TITLE
Mhairi2

### PR DESCRIPTION
**Pull request by mhairimca**
Changed welcome page
Merged with "How UMA works" (cut down version)

### CODE REVIEW

File: docs/getting-started/welcome.md [hunk 1]
```diff
@@ -1,36 +1,55 @@
...
+UMA stands for "Universal Market Access" 
+
+<!--taken from UMA/TRS Mission and Values -->
+Our mission is to make financial markets universally fair and accessible.
+
+We are driven by a shared belief that financial markets should be universally accessible - unrestrained by censorship, your pre-existing financial and social capital, or your beliefs and values. The economic freedom created by open market access enables everyone equal access and footing to pursue prosperity and build their financial independence.  Free and universal access to financial markets is a prerequisite to this free, open and fair financial system.  
+
+Our goal is to enable anyone to attain or transfer any form of risk seamlessly through the UMA protocol thus empowering everyone to participate in this new universally accessible financial system.
+<!--end-->
+
```
Comment (Tommy1231232) [inline docs/getting-started/welcome.md:16]: Recommend adding a "What is UMA" sectioin here

File: docs/getting-started/welcome.md [hunk 2]
```diff
@@ -1,36 +1,55 @@
...
+UMA builds open source infrastructure which enables developers to create financial instruments in a permissionless, secure and efficient manner.  We work with developers, financial engineers and product designers to support them build products on our infrastructure, while supporting tokenholders to actively participate in the governance of our protocol. 
+
+Specifically this involves 
+ - [Priceess financial contract templates](getting-started/priceless_contracts.md) which can be used to create synthetic tokens
+ - [Governance](getting-started/governance.md) which manages improvements to the UMA protocol
```
Comment (Tommy1231232) [inline docs/getting-started/welcome.md:21]: Recommend putting Governance after DVM. I'd also like to tie Governance into "What UMA is" a bit more if we are going to call it out here. Personally, I don't think we need to call out Governance at this point. It's too in the details for a high level overview of UMA.

File: docs/getting-started/welcome.md [hunk 3]
```diff
@@ -1,36 +1,55 @@
...
+ - [Data Verification Mechanism](getting-started/oracle.md) (DVM), an optimistic oracle service
```
Comment (Tommy1231232) [inline docs/getting-started/welcome.md:22]: Recommend adding a bit more detail here to tie the DVM to the priceless contracts. Maybe something like "an optimistic oracle service used to manage and enforce priceless financial contract templates

File: docs/getting-started/welcome.md [hunk 4]
```diff
@@ -1,36 +1,55 @@
...
+
+We are driven by a shared belief that financial markets should be universally accessible - unrestrained by censorship, your pre-existing financial and social capital, or your beliefs and values. The economic freedom created by open market access enables everyone equal access and footing to pursue prosperity and build their financial independence.  Free and universal access to financial markets is a prerequisite to this free, open and fair financial system.  
+
+Our goal is to enable anyone to attain or transfer any form of risk seamlessly through the UMA protocol thus empowering everyone to participate in this new universally accessible financial system.
+<!--end-->
+
+UMA builds open source infrastructure which enables developers to create financial instruments in a permissionless, secure and efficient manner.  We work with developers, financial engineers and product designers to support them build products on our infrastructure, while supporting tokenholders to actively participate in the governance of our protocol. 
+
+Specifically this involves 
+ - [Priceess financial contract templates](getting-started/priceless_contracts.md) which can be used to create synthetic tokens
```
Comment (Tommy1231232) [inline docs/getting-started/welcome.md:20]: Change Priceess to Priceless

File: docs/getting-started/welcome.md [hunk 5]
```diff
@@ -1,36 +1,55 @@
...
+ - [Governance](getting-started/governance.md) which manages improvements to the UMA protocol
+ - [Data Verification Mechanism](getting-started/oracle.md) (DVM), an optimistic oracle service 
+
+Together these enable the creation of fast efficient and secure synthetic derivatives on the ethereum blockchain.
+
+
+**How UMA works**
```
Comment (Tommy1231232) [inline docs/getting-started/welcome.md:27]: Did we align on this being in the welcome page? I'm open to it but I want to focus more on the What UMA does a bit more in that case. For example, We should explain our priceless concept a bit more where we talk about our priceless contract template. Or we put this in a different section after the level 2 of Priceless contracts and DVM

File: docs/getting-started/welcome.md (hunk 4 above)
Comment (mhairimca) [reply docs/getting-started/welcome.md:20]: done

File: docs/getting-started/welcome.md (hunk 2 above)
Comment (mhairimca) [reply docs/getting-started/welcome.md:21]: added more info in text and re-ordered the list.

**Pull request closed without merging**